.github/workflows/deployment.yml

```
@@ -0,0 +1,62 @@
+# Copied from https://github.com/notlmn/browser-extension-template/blob/master/.github/workflows/deployment.yml
+name: Deployment
+
+on:
+  workflow_dispatch:
+  # You can manually trigger a deployment on GitHub.com
+  # https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
+  schedule:
+    - cron: '31 13 * * 3'
+
+jobs:
+  Build:
+    outputs:
+      created: ${{ steps.daily-version.outputs.created }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 20
+      - run: npm install
+      - run: npm test # This includes the build
+      - uses: fregante/daily-version-action@v1
+        name: Create tag if necessary
+        id: daily-version
+      - uses: notlmn/release-with-changelog@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          template: '{commits}'
+          exclude: '^Meta|^Document|^Readme|^Lint'
+      - name: Update extension’s meta
+        env:
+          VER: ${{ steps.daily-version.outputs.version }}
+        run: |
+          echo https://github.com/$GITHUB_REPOSITORY/tree/$VER > distribution/SOURCE_URL.txt
+          npm run version
+      - uses: actions/upload-artifact@v2
+        with:
+          path: distribution
+
+  Chrome:
+    needs: Build
+    if: github.event_name == 'workflow_dispatch' || needs.Build.outputs.created
```

Maybe the first condition can be merged into Build’s output, so instead of exporting .created we can exported a .shouldDeploy and check that here
```suggestion
    if: needs.Build.outputs.shouldDeploy
```

---

I've modified this as [d09f53a5](https://github.com/sindresorhus/hide-files-on-github/pull/100/commits/d09f53a515e400fc1c7e859e68c240d94e018f2e#diff-082c28d748ad2e3eecc5508d740d9417R14), let me know if that works.